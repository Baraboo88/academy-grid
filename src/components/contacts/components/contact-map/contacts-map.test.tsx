import * as renderer from 'react-test-renderer';
import * as React from 'react';

import 'jest-styled-components';

import ContactsMap from './contacts-map';

it(`ContactMap successfully rendered`, () => {

  const tree = renderer.create(
    <ContactsMap />);
  expect(tree).toMatchSnapshot();
});

