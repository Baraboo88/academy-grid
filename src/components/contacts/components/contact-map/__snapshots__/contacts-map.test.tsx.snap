// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactMap successfully rendered 1`] = `
<div
  style={
    Object {
      "height": "100%",
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": "100%",
    }
  }
/>
`;
